airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -200,3 +202,61 @@ def delete_file(self, container_name, blob_name, is_prefix=False, ignore_if_miss
         for blob_uri in blobs_to_delete:
             self.log.info("Deleting blob: %s", blob_uri)
             self.connection.delete_blob(container_name, blob_uri, delete_snapshots='include', **kwargs)
+
+    def get_blob_to_stream(
+        self,
+        container_name,
+        blob_name,
+        stream,
+        snapshot=None,
+        start_range=None,
+        end_range=None,
+        validate_content=False,
+        progress_callback=None,
+        max_connections=2,
+        lease_id=None,
+        if_modified_since=None,
+        if_unmodified_since=None,
+        if_match=None,
+        if_none_match=None,
+        timeout=None,
+    ):
+        """
+
+        :param container_name:
+        :param blob_name:
+        :param stream:
+        :param snapshot:
+        :param start_range:
+        :param end_range:
+        :param validate_content:
+        :param progress_callback:
+        :param max_connections:
+        :param lease_id:
+        :param if_modified_since:
+        :param if_unmodified_since:
+        :param if_match:
+        :param if_none_match:
+        :param timeout:
+        :return:
+        """
+        return self.connection.get_blob_to_stream(
+            container_name,
+            blob_name,
+            stream,
+            snapshot,
+            start_range,
+            end_range,
+            validate_content,
+            progress_callback,
+            max_connections,
+            lease_id,
+            if_modified_since,
+            if_unmodified_since,
+            if_match,
+            if_none_match,
+            timeout,
+        )
+
+    def download(self, offset, length, **kwargs):
+        return self.connection.download_blob(offset=None, length=None, **kwargs)
```

Comment:
```suggestion
        return self.connection.download_blob(offset=offset, length=length, **kwargs)
```

Reply:
That method was experimental, won't be in the final commit.